internal/xds/translator/authentication.go

```diff
@@ -195,34 +195,25 @@ func buildClusterFromJwks(jwks *jwksCluster) (*cluster.Cluster, error) {
 		return nil, errors.New("jwks is nil")
 	}
 
-	endpoints, err := jwks.toLbEndpoints()
+	destinations, err := jwks.toRouteDestinations()
 	if err != nil {
 		return nil, err
 	}
 
+	cluster := buildXdsCluster(jwks.name, destinations, false /*isHTTP2 */)
+	cluster.ConnectTimeout = durationpb.New(10 * time.Second)
```

Comment:
@arkodg I followed the rate limit cluster connection timeout settings for the JWKS cluster. Should both clusters stick with `ConnectTimeout` setting from buildXdsCluster()?

Reply:
@sbko can all common settings such as `ConnectTimeout`  live inside `buildXdsCluster` ? the goal is to not have the caller set these fields unless they want to override it

Reply:
@arkodg `ConnectTimeout` already set inside `buildXdsCluster`, but default value is 5 seconds. should I change default value to 10 seconds?

Reply:
I think 5 is fine. I dont think the ratelimit or auth cluster callers need to perform any overrides on the returned cluster val.
Another thing you will need to handle is
```
ClusterDiscoveryType: &cluster.Cluster_Type{Type: cluster.Cluster_STRICT_DNS}
```
if the endpoints are not static IPs we need to set above field along with other DNS settings mentioned in this PR
the easiest way to do it now would be to extend `buildXdsCluster` and add a `bool isStatic` for now, and set that for the auth and ratelimit callers.